src/models/testSuite.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) AcademyJS and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const testCaseSchema = new Schema({
+    title: {
+        type: String,
+        maxlength: 1024,
+        trim: true,
+        required: true,
+    },
+    description: {
+        type: String,
+        maxlength: 2048,
+        trim: true,
+    },
+});

```

Comment:
Why is this still here? :/ Also, let's rename `testCase` to `test` everywhere because that's the nomenclature Mocha uses.